hypha/apply/funds/tests/test_views.py

```diff
@@ -207,6 +207,79 @@ def test_can_create_project(self):
         self.assertTrue(hasattr(submission, 'project'))
         self.assertEquals(submission.project.id, project.id)
 
+    def test_assign_reviewers_primary_action(self):
```

Comment:
I've changed the button assertions to use BeautifulSoup (that definitely makes more sense than updating all of the html) and I've added tests for applicant and reviewer views. With the staff view, I've split the existing primary action test into can see and can't see. I'm not sure that there's going to be that much value in splitting the tests further than that.

Reply:
I think you could split it down, i like to maintain separation so that we dont cross contaminate and can see which state do/dont work rather than bailing early from the assertions.

It works though, approving.